website/docs/BoilerplateProjects.md

```diff
@@ -52,6 +52,20 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
   - Page Object Pattern
   - Sauce Labs integration
 
+## [17thSep/WebdriverIO_Master](https://github.com/17thSep/WebdriverIO_Master)
+
+   Boilerplate project to run **WebdriverIO v7** tests for the following <br>
+   
+   [1. Cucumber](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Cucumber)               - WDIO 7 scripts with TypeScript in Cucumber Framework<br>
+   [2. Mocha](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Mocha)           - WDIO 7 scripts with TypeScript in Mocha Framework<br>
+   [3. Docker](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Docker)         - Run WDIO 7 script in Docker<br>
+   [4. Networklogs](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/NetworkLogs)     - Capture Network logs<br>
+   [5. BasicTypeScript](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/BasicTypeScript) - Only TypeScript<br>
+   
+   More to follow...
+
+   ### You can also [watch them created here](https://www.youtube.com/channel/UCqaDA1xslraCbam2CxuKhUw)<br>
```

Comment:
could you remove this? it is not related to boilerplate projects

Reply:
ok